packages/expo-asset/plugin/src/utils.ts

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs/promises';
+import path from 'path';
+
+export const imageTypes = ['.png', '.jpg', '.gif'];
+export const fontTypes = ['.otf', '.ttf'];
+export const mediaTypes = ['.mp4', '.mp3'];
+export const ACCEPTED_TYPES = ['.json', '.db', ...imageTypes, ...mediaTypes, ...fontTypes];
```

Comment:
Why different casing? 😅

Reply:
Good question 😅 couldn't decide which I preferred and then forgot to change it 😄